pkgs/tools/networking/ppp/default.nix

```diff
@@ -1,42 +1,27 @@
 { lib, stdenv, fetchurl, fetchpatch, fetchFromGitHub, substituteAll, libpcap, openssl }:
 
 stdenv.mkDerivation rec {
-  version = "2.4.8";
+  version = "2.4.9";
   pname = "ppp";
 
   src = fetchFromGitHub {
     owner = "paulusmack";
     repo = "ppp";
     rev = "ppp-${version}";
-    sha256 = "1i88m79h6g3fzsb4yw3k8bq1grsx3hsyawm7id2vcaab0gfqzjjv";
+    sha256 = "sha256-8+nbqRNfKPLDx+wmuKSkv+BSeG72hKJI4dNqypqeEK4=";
   };
 
   patches =
     [
```

Comment:
```suggestion
  patches = [
```

Reply:
When will you fix `nixos-generate-config`, which does exactly the same in hardware-configuration.nix?